runelite-client/src/main/java/net/runelite/client/plugins/camera/CameraPlugin.java

```diff
@@ -66,6 +71,11 @@
 	private static final String LOOK_SOUTH = "Look South";
 	private static final String LOOK_EAST = "Look East";
 	private static final String LOOK_WEST = "Look West";
+	private static final int RESET_DIRECTION_TIMEOUT = 5;
+	private int lastUpdatedTick = 0;
+	private static final String defaultLeftClickOption = LOOK_NORTH;
```

Comment:
This should be moved with the other constants and renamed to use the same naming scheme.
```suggestion
	private static final String DEFAULT_LEFT_CLICK_OPTION = LOOK_NORTH;
```